src/docfx/build/page/BuildPage.cs

```diff
@@ -43,7 +43,13 @@ internal static class BuildPage
             if (error != null)
                 errors.Add(error);
 
-            return (errors, model, dependencies.Build());
+            var output = (object)model;
+            if (!file.Docset.Config.Output.Json && schema.Attribute is PageSchemaAttribute)
+            {
+                output = await Template.Render(model);
+            }
+
+            return (errors, output, dependencies.Build());
```

Comment:
>output [](start = 28, length = 6)

this can be PageModel or string, but `Build.cs` always uses `context.WriteJson` to write it. Add a `context.WriteString` when it's string?